### PR TITLE
Feature footer

### PR DESCRIPTION
**Pull request by illja-popovics**
Add footer markup with CSS styling. Used partials. Also created a new js file for subscription logic and imported it via index_scripts.html

### CODE REVIEW

File: src/js/subscription.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    const email = subscribeForm.elements['email'].value;
+
+    try {
+      subscribeBtn.disabled = true;
+      subscribeBtn.textContent = 'Subscribing...';
+
+      const response = await servicePostSubscription(email);
+      console.log('Subscription successful:', response);
+
+      alert('Subscription successful!');
```
Comment (kazamov) [inline src/js/subscription.js]: Use iziToast, we already have an example in the code

File: src/js/subscription.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    } catch (error) {
+      console.error('Subscription failed:', error);
```
Comment (kazamov) [inline src/js/subscription.js]: Use iziToast, we already have an example in the code

File: src/js/subscription.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+      alert('Subscription failed. Please try again.');
```
Comment (kazamov) [inline src/js/subscription.js]: Remove alert, and do not log

File: src/js/subscription.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    event.preventDefault();
+
+    const email = subscribeForm.elements['email'].value;
+
+    try {
+      subscribeBtn.disabled = true;
+      subscribeBtn.textContent = 'Subscribing...';
+
+      const response = await servicePostSubscription(email);
+      console.log('Subscription successful:', response);
```
Comment (kazamov) [inline src/js/subscription.js]: Remove log

File: src/css/footer.css [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+  display: flex;
+  gap: 12px;
+}
+
+@media screen and (min-width: 768px) {
+  .footer-socials-list {
+  }
+}
+
+.socials-link {
```
Comment (kazamov) [inline src/css/footer.css:53]: We already have social links on the black background in the mobile menu. Please move those styles to common place, or use the same markup. Thus you can remove your own css styles

**Review by kazamov — CHANGES_REQUESTED** [review]

File: src/css/footer.css [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+  border: 1px solid var(--light);
+  background-color: var(--dark);
+
+  color: rgba(244, 244, 244, 0.6);
+  transition: all var(--transition);
+  color: var(--light);
+}
+
+.subscribe-form-input:focus,
+.subscribe-form-input:hover {
```
Comment (kazamov) [inline src/css/footer.css]: Remove hover selector

File: src/css/footer.css [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+  outline: none;
```
Comment (kazamov) [inline src/css/footer.css]: Use `outline: 1px solid var(--light)`. Remove 'border-color: orange'. We already have an input on the white background and it has similar styling but the outline is --dark. The would be if just extend the existing input component with dark version and reuse most of the stylings. Styles are in the search-input.css file.

**Review by kazamov — CHANGES_REQUESTED** [review]

File: src/js/subscription.js (hunk 3 above)
Comment (illja-popovics) [reply src/js/subscription.js]: done

File: src/js/subscription.js (hunk 2 above)
Comment (illja-popovics) [reply src/js/subscription.js]: Done, added messages via izitoast

File: src/js/subscription.js (hunk 1 above)
Comment (illja-popovics) [reply src/js/subscription.js]: Done, added messages via izitoast

File: src/js/subscription.js (hunk 4 above)
Comment (illja-popovics) [reply src/js/subscription.js]: done

File: src/css/footer.css (hunk 5 above)
Comment (illja-popovics) [reply src/css/footer.css:53]: Used styles from the header, but still needed to leave footer styles due to different logic with adaptive versions

File: src/css/footer.css (hunk 6 above)
Comment (illja-popovics) [reply src/css/footer.css]: done

File: src/css/footer.css (hunk 7 above)
Comment (illja-popovics) [reply src/css/footer.css]: done

**Comment by illja-popovics** [comment]
Hi @kazamov  I have fixed the code based on your suggestion. Please take a look and approve if you have time

**Review by kazamov — APPROVED** [review]

**Pull request merged**